engine/conditions.go

```diff
@@ -7,6 +7,7 @@ type CondtionResponse struct {
 	// For each time the condition is hit, add all of the context.
 	// keys here, will be used in the message.
 	ConditionHitContext []map[string]string
+	Filepaths           []string
```

Comment:
I was actually thinking that this would be in the condition hit context

Reply:
It's a bit awkward to fit in that structure, unless we want to make it a map[string]interface{}. I think we should maybe have a broader talk about pipelining that will relate to this though

Reply:
I had it there originally but it was just a single entry in the list where each file was a key and the value didn't matter, but I wasn't sure how to communicate that the entry itself was filepaths, unless the key should be `Filepaths` and the value a serialized string containing the list of results?

Reply:
Ah nvm I think I misinterpreted that structure, I think I see what you mean

Reply:
Though I think it may make sense to have a set of structured fields in addition to the unstructured ones for basic pipelining